diff --git a/lib/index.js b/lib/index.js
index c2308ac1d57749208febab24af84bad396aa14a9..93295d62746528101ecc2285302dae3a699858a7 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -29,8 +29,8 @@ __exportStar(require("./api"), exports);
 __exportStar(require("./command"), exports);
 __exportStar(require("./creator"), exports);
 __exportStar(require("./server"), exports);
-__exportStar(require("./node/creator"), exports);
-__exportStar(require("./node/util"), exports);
+// __exportStar(require("./node/creator"), exports);
+// __exportStar(require("./node/util"), exports);
 __exportStar(require("./util/bitfield"), exports);
 __exportStar(require("./util/collection"), exports);
 __exportStar(require("./rest/DiscordHTTPError"), exports);
@@ -41,7 +41,7 @@ __exportStar(require("./servers/bun"), exports);
 __exportStar(require("./servers/cfworker"), exports);
 __exportStar(require("./servers/lambda"), exports);
 __exportStar(require("./servers/azure"), exports);
-__exportStar(require("./servers/express"), exports);
+// __exportStar(require("./servers/express"), exports);
 __exportStar(require("./servers/fastify"), exports);
 __exportStar(require("./servers/gateway"), exports);
 __exportStar(require("./servers/gcf"), exports);
